package.json

```diff
@@ -14,5 +14,8 @@
     "server/node_modules"
   ],
   "author": "",
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
+    "moment": "^2.24.0"
```

Comment:
moment should not be added in the dependencies of the main package.json. Are you using it in the client or in the server? You need to put this dependency in the right package.json (in client or server) and remove it from here.

Reply:
Added moment to the client side, removed from main